Sentinel.podspec

```diff
@@ -62,6 +62,11 @@ Pod::Spec.new do |s|
     sp.source_files = 'Sentinel/Classes/CrashDetection/**/*'
     sp.dependency 'Sentinel/Core'
   end
+
+  s.subspec 'Database' do |sp|
+    sp.source_files = 'Sentinel/Classes/DatabaseImportExport/**/*'
```

Comment:
I would suggest changing path to use `DatabaseTool` instead of `DatabaseImportExport`.